easybuild/easyconfigs/b/bgen/bgen-1.0.2-iomkl-2018a.eb

```diff
@@ -0,0 +1,27 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'Waf'
+name = 'bgen'
+version = '1.0.2'
+
+homepage = "https://bitbucket.org/gavinband/bgen"
+description = """A reference implementation of the BGEN format in C++, originally sourced from
+ the qctool implementation. In addition three utilities - bgenix, which provides indexed access
+ to BGEN files, cat-bgen which efficiently concatenates BGEN files, and edit-bgen which is used
+ to manipulate BGEN metadata - are provided. See the wiki for documentation on these programs."""
+
+toolchain = {'name': 'iomkl', 'version': '2018a'}
+
+sources = ['32abb11ef9dd.tar.gz']
+source_urls = ['https://bitbucket.org/gavinband/bgen/get/']
```

Comment:
It might be preferable to make use of the filename/download_filename syntax for sources to give the downloaded file a name that matches the version in the easyconfig file. See https://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html?highlight=sources%20specification#alternative-formats-for-sources

And the same for qctool

Reply:
And add the missing checksums.